spec/process-manager-spec.js

```diff
@@ -38,7 +38,7 @@ describe('ProcessManager', () => {
     processManager.killChildProcesses()
     processManager.executeChildProcess(constructCommand('new.tex'), { allowKill: true }).then(result => { newKilled = true })
 
-    waitsFor(() => oldKilled, 5000)
+    waitsFor(() => oldKilled, 10000)
```

Comment:
Let's try removing these; I'm hoping whatever the underlying issue on AppVeyor was, that is has been resolved by now. Either way, your changes should not cause any errors in these tests, so if these are still timing out, that should be addressed separately.

Reply:
Done!